src/java/org/apache/cassandra/streaming/StreamSession.java

```diff
@@ -728,6 +739,12 @@ private void prepareAsync(Collection<StreamRequest> requests, Collection<StreamS
             maybeCompleted();
     }
 
+    private void setComplete()
+    {
+        state(State.WAIT_COMPLETE);
```

Comment:
I add WAIT_COMPLETE just to be consistent, I don't see any logic where this matters; its just a consistency thing.